tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 # when changing this list, adapt CONTRIBUTING.md accordingly:
```

Comment:
Does this not count as "supporting" 3.10? If it is you should add the new version to CONTRIBUTING.md per the comment.